packages/drivers/routerlicious-driver/src/deltaStorageService.ts

```diff
@@ -75,31 +72,12 @@ export class DeltaStorageService implements IDeltaStorageService {
         id: string,
         from: number,
         to: number): Promise<IDeltasFetchResult> {
-        const query = querystring.stringify({ from, to });
-
-        const headers: OutgoingHttpHeaders = {
-            "x-correlation-id": uuid(),
-        };
-
-        const storageToken = await this.tokenProvider.fetchStorageToken(
-            tenantId,
-            id,
-        );
-
-        if (storageToken) {
-            headers.Authorization = `Basic ${storageToken.jwt}`;
-        }
-
-        const ops = await Axios.get<ISequencedDocumentMessage[]>(
-            `${this.url}?${query}`, { headers });
-
-        this.logger.sendTelemetryEvent({
-            eventName: "R11sDriverToServer",
-            correlationId: headers["x-correlation-id"] as string,
```

Comment:
I removed this log as I have not heard/seen of anyone using it so far. Instead, correlationId is logged on CriticalRequestError (i.e. unknown error on request). I will revisit adding correlationId to request telemetry at a later date.